### ISSUE TITLE
Error when using different units with bielliptic maneuver

### ISSUE THREAD

**Issue by Sedictious** [poliastro/poliastro#731]
🐞 **Problem**

There seems to be a bug when using different units in bielliptic maneuver.

🖥 **How to reproduce this bug**

* In ``constants.py`` replace the definition of ``Earth`` with: 

``` python
R_earth = Constant(
    "R_earth",
    "Earth equatorial radius",
    6.3781366e3,
    "km",
    0.0001,
    "IAU Working Group on Cartographic Coordinates and Rotational Elements: 2015",
    system="si",
)
```

After running the tests in ``test_maneuver.py``, ``test_bielliptic_maneuver()`` should fail:
```
actual = <Quantity 653697.65245305 s>, desired = <Quantity 593.919803 h>
rtol = 1e-06, atol = None, kwargs = {}
np = <module 'numpy' from 'C:\\Users\\P\\Desktop\\poliastro\\venv\\lib\\site-packages\\numpy\\__init__.py'>
_unquantify_allclose_arguments = <function _unquantify_allclose_arguments at 0x0000027155408F28>

    def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                 **kwargs):
        """
        Raise an assertion if two objects are not equal up to desired tolerance.
    
        This is a :class:`~astropy.units.Quantity`-aware version of
        :func:`numpy.testing.assert_allclose`.
        """
        import numpy as np
        from astropy.units.quantity import _unquantify_allclose_arguments
        np.testing.assert_allclose(*_unquantify_allclose_arguments(
>           actual, desired, rtol, atol), **kwargs)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-06, atol=0
E       
E       Mismatch: 100%
E       Max absolute difference: 1484413.63834695
E       Max relative difference: 0.69426397
E        x: array(653697.652453)
E        y: array(2138111.2908)
```

(Tested with Python 3.6)

💡 **Possible solutions**

If this bug is reproduce-able, it may be the case that somewhere the astropy units are converted into raw values, so fixing this may fix the error.

**Comment by jorgepiloto** [comment]
Investigating this... Could it be the fact that you wrote the number in `km` and variable `system='si'` is set to the international system?

**Comment by Sedictious** [comment]
@jorgepiloto Yes it seems to work when set to meters (this also seems to be the only test that fails).

**Comment by Juanlu001** [comment]
I can confirm that the test fails, and also that _it's the only test that fails_, which suggests that the unit definition that @Sedictious proposes is likely correct. This is most likely a bug.

**Comment by Juanlu001** [comment]
I confirm this was fixed in #744! :tada:
